camayoc/tests/qpc/ui/test_credentials.py

```diff
@@ -0,0 +1,79 @@
+# coding=utf-8
+"""Tests for handling sources in the UI.
+
+:caseautomation: automated
+:casecomponent: ui
+:caseimportance: high
+:caselevel: integration
+:requirement: Sonar
+:testtype: functional
+:upstream: yes
+"""
+from uuid import uuid4
+
+from selenium.common.exceptions import NoSuchElementException
+
+from smartloc import Locator
+
+from widgetastic.exceptions import WidgetOperationFailed as wof
+from widgetastic.widget import Checkbox
+
+from widgetastic_patternfly import Button, Dropdown
+
+from .utils import clear_toasts, fill
+from .views import CredentialModalView, DashboardView, DeleteModalView
+
+
+def create_credential(view):
+    """Create a credential through the UI."""
+    dash = DashboardView(view)
+    dash.nav.select('Credentials')
+    # TODO: Parametrize for different types of credentials.
```

Comment:
Make sure we have an issue to not forget about this :smile:

Reply:
This is part of the issue and will be in the next PR :)